pages/Nvidia/_index.md

```diff
@@ -152,109 +147,120 @@ env = ELECTRON_OZONE_PLATFORM_HINT,auto
+will need to remove the `optimus-manager` package if installed (disabling the
+service does not work). You also need to change your BIOS settings from hybrid
+graphics to discrete graphics.
 
-Make a new file at `/etc/modprobe.d/nvidia.conf` and paste this in:
+### Flickering in XWayland games
 
-```conf {filename="/etc/modprobe.d/nvidia.conf"}
-options nvidia NVreg_RegistryDwords="PowerMizerEnable=0x1; PerfLevelSrc=0x2222; PowerMizerLevel=0x3; PowerMizerDefault=0x3; PowerMizerDefaultAC=0x3"
-```
+XWayland games may flicker or present frames out-of-order in a way which makes
+them unplayable. This is due to the lack of implicit synchronization in the
+driver, and/or flaky explicit sync support in newer ones.
 
-Reboot your computer and it should be working.
+There are a few fixes:
 
-If it does not, try:
+1. Install the latest versions of `xorg-xwayland`, `wayland-protocols` and
+   Nvidia driver. Ensure `xorg-xwayland` is at least version 24.1,
+   `wayland-protocols` is at least version 1.34 and Nvidia driver is at least
+   version 555. These enable explicit sync on the Nvidia driver and should avoid
+   flickering.
 
-1. Lowering your monitor's refresh rate: This can stop the flickering
-   altogether.
-2. Using the [Nouveau driver](https://wiki.archlinux.org/title/Nouveau) as
-   mentioned above.
+2. If your GPU is no longer supported by the 555 or later drivers, install older
+   Nvidia drivers which do not exhibit this issue. The last ones which would
+   work will be the 535xx series of drivers. These can be installed on Arch via
+   [these AUR packages](https://aur.archlinux.org/packages?O=0&K=535xx)
 
 ### Suspend/wakeup issues
 
-Enable the services `nvidia-suspend.service`, `nvidia-hibernate.service` and
-`nvidia-resume.service`, they will be started by systemd when needed.
+On Arch Linux and NixOS, the below is already done for you, but for others:
+
+- Enable the services `nvidia-suspend.service`, `nvidia-hibernate.service` and
+  `nvidia-resume.service`, they will be started by systemd when needed.
 
 Add `nvidia.NVreg_PreserveVideoMemoryAllocations=1` to your kernel parameters if
 you haven't already.
 
-{{< callout >}}
-
-As previously mentioned, suspend functions are currently broken on `nvidia-open-dkms`
-[due to a bug](https://github.com/NVIDIA/open-gpu-kernel-modules/issues/472), so
-make sure you're on `nvidia-dkms`.
-
-{{< /callout >}}
-
 For Nix users, the equivalent of the above is
 
 ```nix {filename="configuration.nix"}
-boot.kernelParams = [ "nvidia.NVreg_PreserveVideoMemoryAllocations=1" ];
-
 hardware.nvidia.powerManagement.enable = true;
-
-# Making sure to use the proprietary drivers until the issue above is fixed upstream
-hardware.nvidia.open = false;
 ```
 
+{{< callout >}}
+
+According to Nvidia, suspend / wakeup issues should be solved on the Nvidia open
+drivers. If it still doesn't work and you're using the open drivers, it may be
+worth trying the proprietary ones, just in case.
```

Comment:
```suggestion
According to Nvidia, suspend/wakeup issues should be solved on the Nvidia open
driver. If it still doesn't work and you're using the open driver, it may be
worth trying the fully proprietary one.
```